src/oss/confluent.sh

```diff
@@ -1433,6 +1489,52 @@ version_command() {
     fi
 }
 
+produce_usage() {
+    cat <<EOF
+Usage: ${command_name} <topicname> [--value-format avro] [other optional args]
```

Comment:
"other optional args" is a bit vague if you dont know the underlying tool.
Maybe tell the user to pass --help to get a full list of options (and just pass that on to the underlying tool),
do note however that the tool may differ depending on --value-format avro..

Reply:
Unfortunately, this won't work due to how kafka-console-producer works

```bash
confluent produce t3 --help
This CLI is intended for development only, not for production
https://docs.confluent.io/current/cli/index.html

help is not a recognized option
```